boost-maven-plugin/src/main/java/boost/project/JDBCBoosterPackConfigurator.java

```diff
@@ -0,0 +1,61 @@
+package boost.project;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+
+public class JDBCBoosterPackConfigurator implements BoosterPackConfigurator {
```

Comment:
I may be missing something, but where is the BoosterPackConfigurator interface?

Reply:
in boost-common